### DISCUSSION TITLE
Is there a document that tells what EVERY setting, button, and slider does ?

### DISCUSSION THREAD

**Discussion by swamp-cabbage** [vladmandic/sdnext#2981] [Q&A]
I have looked at changelog, readme, wiki - none of them go into specific detail with regard to what every setting does. Is this a big mystery that we are responsible for solving with trial and error, or is there a "user manual" for SD.Next (specifically diffusers pipeline) that I have yet to discover ? Thanks !

**Comment by Ugijago** [comment]
if you use nvidia its should be easy , but for amd owner gpu like me its super mystery , even many tutorial is not really solved the problem i have, and many tutorial is not work :( even now i am still searching how to use onnx olive to run always have error, well i will continue searching :)

**Comment by vladmandic** [comment]
this is a community based project. best course is
1. ask specific question on discord
2. once you receive answer, document it so others can benefit from it in the future.

if everyone expects everything to be ready ahead of time, docs/hints never improve on their own.
